docs/reference/configuration.txt

```diff
@@ -541,9 +541,8 @@ Place the following in one of environment configuration files, such as
 
 .. note::
 
-  The ``log_level`` Mongoid `configuration option <configuration-options>`_
-  is not used when Mongoid operates in a Rails application, because Mongoid
-  inherits Rails' log level in this case.
+  The ``log_level`` Mongoid configuration option is not used when Mongoid operates
+  in a Rails application, because Mongoid inherits Rails' log level in this case.
```

Comment:
Q: Is this link no longer needed?

Reply:
I couldn't find any other page with info on Mongoid configuration options or the `log_level` option. Since this page itself (configuration.txt) has the most specific information about configuration options, I didn't think it made sense to link to somewhere else.